src/page-modules/contact/components/input/description-modal.tsx

```diff
@@ -43,7 +43,7 @@ const DescriptionModal = ({
       document.removeEventListener('keydown', handleEscapeOrClickOutside);
       document.removeEventListener('mousedown', handleEscapeOrClickOutside);
     };
-  }, []);
+  });
```

Comment:
Why is this changed and why remove the empty dependency list?

Reply:
I received a warning when the deployment ran `yarn tsc` during deployment. I don't think it makes any difference if I remove  the empty dependency list or not, but I can undo the change to keep order.

Reply:
We use empty dependency arrays for useEffects if we want the useEffect to run once (on initial render). Else, you want to add the dependencies to the array so that it runs when they changes (i.e. `handleEscapeOrClickOutside` in this case). I guess the warning you get is from es lint and not typescript, but you can disable it using `// eslint-disable-line react-hooks/exhaustive-deps` if you want to use an empty dependency list